archive/frames/filters.py

```diff
@@ -31,11 +52,11 @@ def intersects_filter(self, queryset, name, value):
 
     def public_filter(self, queryset, name, value):
         if not value:
-            return queryset.exclude(L1PUBDAT__lt=timezone.now())
+            return queryset.exclude(public_date__lt=timezone.now())
         return queryset
 
     class Meta:
         model = Frame
-        fields = ['DATE_OBS', 'DAY_OBS', 'PROPID', 'OBSTYPE', 'INSTRUME',
-                  'RLEVEL', 'SITEID', 'TELID', 'FILTER', 'L1PUBDAT',
-                  'BLKUID', 'REQNUM']
+        fields = ['proposal_id', 'configuration_type', 'instrument_id',
+                  'reduction_level', 'site_id', 'telescope_id', 'primary_optical_element',
+                  'observation_id', 'request_id']
```

Comment:
Oh, lol looks like Elisabeth found the same thing above

Reply:
yeah I purposely left those out. The start/end should be used for future date queries (the exact ones are much less useful I Think), and I didn't see the need for a start/end public query since there is already a public boolean filter which actually excludes public frames